sqlalchemy_easy_softdelete/handler/sqlalchemy_easy_softdelete.py

```diff
@@ -12,7 +13,10 @@
 
 
 def activate_soft_delete_hook(
-    deleted_field_name: str, disable_soft_delete_option_name: str, ignored_tables: List[IgnoredTable]
+    deleted_field_name: str,
+    disable_soft_delete_option_name: str,
+    ignored_tables: List[IgnoredTable],
+    enabled_tables: Optional[List[EnabledTable]] = None,
```

Comment:
```suggestion
    enabled_tables: List[EnabledTable],
```

Ideally, the signature should be matching with the other similar option `ignored_tables`